.github/workflows/CI-main.yml

```diff
@@ -18,11 +18,12 @@ on:
 
 env:
   XCODE_VERSION: '15.4'
+  INTERNAL_PATCH_NUMBER: 999
```

Comment:
🤔 We could theoretically extend this pattern to also create a build using the next version of Xcode, or any other very-particular collection of settings for an app. Should we make this variable more clear that it's the patch number for the internal beta, so that if we have other internal builds that aren't beta (but are something else) we have a clearer patter for where to go with it?

Reply:
🤔 Maybe I'm missing something but don't we have that already with the `INTERNAL_` prefix? it's pretty general IMO.

Reply:
Oh yeah makes sense for future different internal builds so QA could easily spot the correct internal based on the version. Although I agree on YAGNI for now and we could as well introduce new specific workflows for different `INTERNAL_` builds which would have the correct internal patch version so we reduce the condition logic inside the general workflows.

Reply:
Updated!

Reply:
I initially had `INTERNAL_DEBUG_MENU_PATCH_NUMBER` but felt unnecessarily verbose while we don't have any other builds, but that was the intention! Went with the `INTERNAL_BETA` for the time being!